crates/repository/src/lib.rs

```diff
@@ -326,6 +341,11 @@ impl Repository {
             create_symlink(&directory, &symlink).await?;
             directory
         };
+
+        if USE_DOCKER {
+            return Ok(());
+        }
```

Comment:
I'd rather skip the invocation of this function instead of skipping parts of the function. But this may have to be adjusted as soon as we have a proper naosdk container image

Reply:
We still need to create the symlink, maybe factoring out the symlink creation to a separate funtion instead? FYI: There is already a container image published with the same semantic as the final one, so this is already testable.

Reply:
I split this now into two separate functions.